package.json

```diff
@@ -5,9 +5,6 @@
   "main": "index.js",
   "dependencies": {
     "@hapi/joi": "^15.1.1",
-    "@types/email-templates": "^7.1.0",
```

Comment:
Moved these three to dev dependencies

Types should not be part of the build
Also acorn is a peer dependency of eslint which is a dev dependency (fixed the versino too)